src/main/java/gov/cms/madie/services/CqmConversionService.java

```diff
@@ -22,7 +22,7 @@ public class CqmConversionService {
   public CqmMeasure convertMadieMeasureToCqmMeasure(QdmMeasure measure, String accessToken) {
     // get Libraries
     List<TranslatedLibrary> translatedLibraries =
-        translationServiceClient.getTranslatedLibraries(measure.getCql(), accessToken);
+        translationServiceClient.getTranslatedLibraries(measure.getCql(), accessToken, false);
```

Comment:
@RohitKandimalla Just a question here: why are we using false instead of includeElmWarning flag?

Reply:
getTranslatedLibraries is being used by different work flows, While Converting to CQM Measure we do not give an option to user yet and that is not in cards. So by using false we are doing what we have been doing all these days.

Reply:
@RohitKandimalla Ok thanks for explanation, I guess same thing as in QrdaService?